### ISSUE TITLE
Frozen layers on centerpoint

### ISSUE THREAD

**Issue by enesdorukadastec** [open-mmlab/mmdetection3d#2252]
### Model/Dataset/Scheduler description

Hi, i want ti frooze pts_backbone on centerpoint.

I added pts_backbone(frozen_stages=1) but i got error, how can i solve this problem or how can i freeze layers

### Open source status

- [ ] The model implementation is available
- [ ] The model weights are available.

**Comment by JingweiZhang12** [comment]
We have not supported freezing the specific stages in the point cloud backbone yet. 
You can implement it in the `__init__` of the pts backbone, referring to the image backbone in mmdet.

**Comment by enesdorukadastec** [comment]
Thanks, i implemented freezing section but i dont understand that model.eval() is used before the param.requires_grad = False on mmdetection or mmclassification. for example resnet. Can you explain why model.eval() is used?

**Comment by JingweiZhang12** [comment]
`model.eval` can freeze the `running_mean` and `running_variance` in BN, but `param.requires_grad=False` can not do this.

**Comment by enesdorukadastec** [comment]
thanks @JingweiZhang12

**Comment by enesdorukadastec** [comment]
Thanks @JingweiZhang12

**Comment by Varatharajan-Raja** [comment]
> Thanks, i implemented freezing section but i dont understand that model.eval() is used before the param.requires_grad = False on mmdetection or mmclassification. for example resnet. Can you explain why model.eval() is used?

Can you please share this implementation of freezing the point cloud backbone?
